static/templates/search_operators.hbs

```diff
@@ -0,0 +1,36 @@
+<div class="overlay-modal hide" id="search-operators" tabindex="-1" role="dialog"
+  aria-label="{{t 'Search operators' }}">
+    <div class="modal-body" data-simplebar data-simplebar-auto-hide="false">
+        <div id="operators-instructions">
+            <table class="table table-striped table-condensed table-rounded table-bordered help-table">
+                <thead>
+                    <tr>
+                        <th>{{t "Operator" }}</th>
+                        <th>{{t "Effect" }}</th>
+                    </tr>
+                </thead>
+
+                <tbody>
+                    {{#each search_operator_rows}}
+                    <tr>
+                        {{#if note_html}}
+                        <td colspan="2">{{{note_html}}}</td>
+                        {{else if ignore_operators}}
+                        <td class="operator">{{{operator_key}}}:{{{operator_value}}}</td>
+                        <td class="definition">{{{effect}}}</td>
```

Comment:
Variables to be interpolated as raw HTML with `{{{…}}}` should have a name ending in `_html`, so we know not to put unescaped text into them and cause an XSS vulnerability.

Reply:
Oh! Thanks!